base/client.jl

```diff
@@ -82,17 +82,22 @@ answer_color() = text_colors[repl_color("JULIA_ANSWER_COLOR", default_color_answ
 stackframe_lineinfo_color() = repl_color("JULIA_STACKFRAME_LINEINFO_COLOR", :bold)
 stackframe_function_color() = repl_color("JULIA_STACKFRAME_FUNCTION_COLOR", :bold)
 
-function repl_cmd(cmd, out)
-    shell = shell_split(get(ENV,"JULIA_SHELL",get(ENV,"SHELL","/bin/sh")))
-    shell_name = Base.basename(shell[1])
+function repl_cmd(line::AbstractString, out)
+    if is_windows()
```

Comment:
I think on v0.6, code is conditionally compiled with and without `@static`
So maybe there's no runtime performance difference?

Both `test1()` and `test2()` produce the same llvm:
```
test1() = is_windows() ? 1 : "a"
test2() = @static is_windows() ? 1 : "a"
```
```
julia> @code_llvm test1()

; Function Attrs: uwtable
define i64 @julia_test1_68517() #0 !dbg !5 {
top:
  ret i64 1
}

julia> @code_llvm test2()

; Function Attrs: uwtable
define i64 @julia_test2_68518() #0 !dbg !5 {
top:
  ret i64 1
}
```